mod_cluster/extension/src/main/java/org/wildfly/extension/mod_cluster/ProxyConfigurationResourceDefinition.java

```diff
@@ -343,14 +346,40 @@ public ManagementResourceRegistration register(ManagementResourceRegistration pa
         ManagementResourceRegistration registration = parent.registerSubModel(this);
 
         ResourceDescriptor descriptor = new ResourceDescriptor(this.getResourceDescriptionResolver())
-                .addAttributes(EnumSet.complementOf(EnumSet.of(Attribute.SSL_CONTEXT)))
-                .addExtraParameters(Attribute.SSL_CONTEXT)
-                .addCapabilities(Capability.class);
+                .addAttributes(EnumSet.complementOf(EnumSet.of(Attribute.SIMPLE_LOAD_PROVIDER, Attribute.SSL_CONTEXT)))
+                .addExtraParameters(Attribute.SIMPLE_LOAD_PROVIDER, Attribute.SSL_CONTEXT)
+                .addCapabilities(Capability.class)
+                ;
+        registration.registerReadWriteAttribute(Attribute.SIMPLE_LOAD_PROVIDER.getDefinition(), new OperationStepHandler() {
```

Comment:
You don't need to reimplement the read/write attribute here.  Just delegate to the read/write handler for the actual attribute on the child resource.

Reply:
Done using translation mechanism.